test/integration/custom-domains-test-cases.yaml

```diff
@@ -0,0 +1,74 @@
+config:
+  inherit-env: true
+tests:
+  001 - list domains:
+    command: auth0 domains list
+    exit-code: 0
+  002 - create domain with minimal flags:
+    command: auth0 domains create --domain "integration-test.com" --no-input
+    exit-code: 0
+    stdout:
+      contains:
+        - "ID                       cd_"
+        - "DOMAIN                   integration-test.com"
+        - "STATUS                   pending_verification"
+        - "PROVISIONING TYPE        auth0_managed_certs"
+  003 - unsuccessfully create domain with same name:
+    command: auth0 domains create --domain "integration-test.com" --no-input
+    exit-code: 1
+    stderr:
+      contains:
+        - "An unexpected error occurred while attempting to create the custom domain 'integration-test.com': 409 Conflict: The specified custom domain already exists"
+  004 - show domain:
+    command: auth0 domains show $(./test/integration/scripts/get-custom-domain-id.sh) --no-input
+    exit-code: 0
+    stdout:
+      contains:
+        - "ID                       cd_"
+        - "DOMAIN                   integration-test.com"
+        - "STATUS                   pending_verification"
+        - "PROVISIONING TYPE        auth0_managed_certs"
+  005 - update domain minimal flags:
+    command: auth0 domains update $(./test/integration/scripts/get-custom-domain-id.sh) --no-input
+    exit-code: 0
+    stdout:
+      contains:
+        - "ID                       cd_"
+        - "DOMAIN                   integration-test.com"
+        - "STATUS                   pending_verification"
+        - "PROVISIONING TYPE        auth0_managed_certs"
+  006 - update domain maximal flags:
+    command: auth0 domains update $(./test/integration/scripts/get-custom-domain-id.sh) --policy recommended --no-input
+    exit-code: 0
+    stdout:
+      contains:
+        - "ID                       cd_"
+        - "DOMAIN                   integration-test.com"
+        - "STATUS                   pending_verification"
+        - "PROVISIONING TYPE        auth0_managed_certs"
+        - "TLS POLICY               recommended"
+  007 - verify domain:
```

Comment:
Just a nit query here about wording as the assertion still says `pending_verification` vs the test case being `verify domain`. Other than that, looks good

Reply:
Agreed. Removed because haven't set up a custom domain yet and if we do, this assertion could break. Just being able to call the management API verify endpoint is enough for me IMO.